.github/workflows/ci-x86.yml

```diff
@@ -196,3 +196,66 @@ jobs:
           See more details on github.com/DynamoRIO/dynamorio/actions/runs/${{github.run_id}}
         to: dynamorio-devs@googlegroups.com
         from: Github Action CI
+
+  # Ubuntu22 64-bit Linux build with gcc and run tests:
+  x86-64-ubuntu22:
+    runs-on: ubuntu-22.04
```

Comment:
If this config is same as the other ubuntu one, we can use a run matrix: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
Done.

Reply:
There is some negative interaction between the matrix and our auto-cancel: as soon as the first job is done the 2nd is cancelled.  Reverting the matrix with a comment.